spec/formatting.md

```diff
@@ -590,3 +620,21 @@ rather than the _expression_ in the _selector_ or _pattern_.
 > ```
 > The value is {|horse|}.
 > ```
+
+When there are multiple _declarations_ for the same _variable_,
+the fallback string is formatted based on the _expression_ of
+the first declaration of that _variable_.
```

Comment:
what about if `var` were externally defined? And why is this a good fallback string?

Reply:
> When there are multiple _declarations_ for the same _variable_,

That would be an error.

Reply:
> what about if `var` were externally defined?

Rewrote to include that case as well.

> And why is this a good fallback string?

See https://github.com/unicode-org/message-format-wg/blob/main/spec/formatting.md?plain=1#L208

I wanted to be as consistent as possible with the existing spec, which says that fallback values should be formatted based on the RHS of a declaration. I thought the most consistent thing if there are multiple declarations is to pick one of them. Do you have another suggestion?

Reply:
> > When there are multiple _declarations_ for the same _variable_,
> 
> That would be an error.
> 

@mihnita I'm not sure what you mean -- doesn't there have to be a fallback string to replace any expression that resolves to an error?